.storybook/main.js

```diff
@@ -16,7 +16,14 @@ module.exports = {
     '@storybook/addon-links',
     '@storybook/addon-postcss',
     '@storybook/addon-viewport',
-    '@storybook/addon-storysource',
+    {
+      name: '@storybook/addon-storysource',
+      options: {
+        rule: {
+          test: [/(-story)|(\.stories)\.jsx?$/],
```

Comment:
👏  Awesome — it could make sense to wildcard this for capturing MDX and other potential file formats here too:

```suggestion
          test: /(-story|.stories).*$/,
```

Reply:
Hi Simon, thanks for the suggestion. If we include other files like MDX, a strange little glitch appears at the top of the Layout modules library page:
![image](https://user-images.githubusercontent.com/3118961/136109658-f15c98fe-e52d-43fa-a64c-6309d8451163.png)

I've narrowed it down to the `<Meta>` metadata tag at the top of the page. I'll have to investigate further; let me know if you've got any ideas! Thanks

Reply:
> Hi Simon, thanks for the suggestion. If we include other files like MDX, a strange little glitch appears at the top of the Layout modules library page: ![image](https://user-images.githubusercontent.com/3118961/136109658-f15c98fe-e52d-43fa-a64c-6309d8451163.png)
> 
> I've narrowed it down to the `<Meta>` metadata tag at the top of the page. I'll have to investigate further; let me know if you've got any ideas! Thanks

Oh! I see what you mean — perhaps specific is better in this case? For example, something like the following which only captures JavaScript files:

```suggestion
          test: /(-story|.stories).js$/,
```